### PR TITLE
Add variables to Openshift example

### PR DESCRIPTION
**Pull request by abraverm**

### CODE REVIEW

**Comment by abraverm** [comment]
I've updated the example to one in the documentation and added variables

**Comment by 14rcole** [comment]
[test]

**Comment by 14rcole** [comment]
Assuming upshift tests pass, this looks good to me.  Does this fix the error that you were getting in #814?

**Comment by abraverm** [comment]
It fixed the #814 issue but I don't know why the test fail now...

**Comment by 14rcole** [comment]
It looks like jenkins is failing to build the linchpin command.  Try to rebase changes from the latest develop branch.  There was a versioning fix in there that should make the tests pass again

**Comment by skatlapa** [comment]
[test]

**Comment by abraverm** [comment]
Now its failing on
```
"msg": "Failed to retrieve requested object: 400 Bad Request",
```
Any idea why?

**Comment by 14rcole** [comment]
None, sorry.  Does it work locally for you?

**Comment by abraverm** [comment]
yes

File: docs/source/examples/workspaces/openshift/PinFile [hunk 1]
```diff
@@ -6,19 +6,40 @@ openshift-new:
       - resource_group_name: test1
         resource_group_type: openshift
         resource_definitions:
-          - name: testsecret
+          - name: openshift
             role: openshift_inline
-            namespace: continuous-infra
+            namespace: "{{ namespace | default('coninuous-infra') }}"
```
Comment (skatlapa) [inline docs/source/examples/workspaces/openshift/PinFile]: @abraverm It is not recognizing the namespace: s/coninuous/continuous

**Comment by 14rcole** [comment]
[test]

**Comment by abraverm** [comment]
@14rcole , the example works locally, I don't know why is it failing on [Bad request](https://jenkins-continuous-infra.cloud.continuous-infra.upshift.redhat.com/blue/organizations/jenkins/ci-linchpin/detail/ci-linchpin/792/pipeline#step-161-log-2096)

**Comment by abraverm** [comment]
[merge]

**Pull request merged**